docs/european-user-consent.mdx

```diff
@@ -91,7 +94,25 @@ Also, within your projects `app.json` file, you have to use the `user_tracking_u
 }
 ```
 
-Once set, rebuild your application.
+#### For Expo projects
+
+```json
+// <project-root>/app.json
+{
+  "expo": {
```

Comment:
Do we still need to show the root "expo" object? I thought this is discouraged starting from SDK 51.

Reply:
Expo still officially supports SDK 50 in which the root "expo" object would still be required. Give that SDK 51 only just released in May, I think a lot of this library's users are still on SDK 50.